packages/clients/diplan/src/plugins/AddressSearch/AddressSearch.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <SearchContainer id="diplan-search-container" />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+// TODO: This import should be changed in the future
+import { SearchContainer } from '@polar/plugin-address-search'
+
+export default Vue.extend({
+  name: 'DiplanAddressSearch',
+  components: {
+    SearchContainer,
+  },
+})
+</script>
+
+<style lang="scss">
+#diplan-search-container {
+  .v-toolbar {
```

Comment:
```suggestion
#diplan-search-container {
  z-index: 1;
  .v-toolbar {
```

This way, the result box stops underlapping the attributions.